docs/11-decorators.md

```diff
@@ -42,6 +37,40 @@ ActiveAdmin.register Post do
 end
 ```
 
+If you wish to use a PORO decorator instead, all you need to do is
+accept the record to be decorated as a parameter in the initializer:
+
+```ruby
+# app/decorators/post_decorator.rb
+class PostDecorator
+  attr_reader :post
+  delegate_missing_to :post
+
+  def initialize(post)
+    @post = post
+  end
+end
+```
+
+You can also customize the way a collection is wrapped by implementing
```

Comment:
After a second thought, and although I believe using duck typing rather that checking from draper directly is fine, I think I wouldn't document this unless there's a good use case for it. The one you mentioned in https://github.com/activeadmin/activeadmin/pull/6249#discussion_r449672211 is probably best implemented as a separate decorator and our decorator support requires decorators to be instantiated by passing the decorated object as the single parameter anyways.

Reply:
Agreed, it felt very weird documenting that part